### PR TITLE
Update istio-mesh-dashboard.json

### PR DESCRIPTION
**Pull request by Kampe**
update link pointers to service and workload dashboards on mesh dashboard.

**Please provide a description of this PR:**
Istio Mesh Dashboard links did not work.

### CODE REVIEW

File: manifests/addons/dashboards/istio-mesh-dashboard.json [hunk 1]
```diff
@@ -1461,7 +1461,7 @@
           "link": false,
           "linkTargetBlank": false,
           "linkTooltip": "$__cell dashboard",
-          "linkUrl": "/dashboard/db/istio-workload-dashboard?var-namespace=${__cell_3:raw}&var-workload=${__cell_2:raw}",
+          "linkUrl": "/d/12010103e01030/istio-workload-dashboard?var-namespace=${__cell_3:raw}&var-workload=${__cell_2:raw}",
```
Comment (zirain) [inline manifests/addons/dashboards/istio-mesh-dashboard.json:1464]: this seems not correct? should be `UbsSZTDik` ? from https://github.com/istio/istio/blob/f611f67505bb60c469f93aca9de0d7f14e8d3538/manifests/addons/dashboards/istio-workload-dashboard.json#L2670

Comment (Kampe) [reply manifests/addons/dashboards/istio-mesh-dashboard.json]: you're right, I had an older version of the dashboard with this uid set, this has been fixed.

**Review by zirain — COMMENTED** [review]
lgtm

**Review by ericvn — DISMISSED** [review]
Question becomes if we want to have a generated uid or something that is more user friendly?

https://grafana.com/docs/grafana/latest/http_api/dashboard/#identifier-id-vs-unique-identifier-uid

I had previously opened an issue when we moved to uids to go back to a simpler link: https://github.com/istio/istio/issues/35226

**Comment by ericvn** [comment]
@istio/wg-policies-and-telemetry-maintainers Thoughts on if we should have the older style links like `dashboard/db/istio-*-dashboard` or names like `d/G8wLrJIZk/istio-mesh-dashboard`. The issue I opened a while ago when we made the switch: https://github.com/istio/istio/issues/35226.

**Comment by zirain** [comment]
I've checked grafana doc, can not find out a way to set url like the old way. maybe some one know grafana better than me can give a better solution.

**Comment by howardjohn** [comment]
Why is it broken? Seems like it used to work...?

**Comment by douglas-reid** [comment]
> Why is it broken? Seems like it used to work...?

https://grafana.com/docs/grafana/v7.5/whatsnew/whats-new-in-v5-0/#dashboard-model-persistent-urls-and-api-changes

> We’re changing the URL format for dashboards from /dashboard/db/:slug to /d/:uid/:slug. We’ll keep supporting the old slug-based URLs for dashboards and redirects to the new one for backward compatibility. Please note that the old slug-based URLs have been deprecated and will be removed in a future release.

@ericvn it looks like we can't go back to old-style.

**Comment by zirain** [comment]
not stale

**Review by zirain — APPROVED** [review]

**Review by howardjohn — APPROVED** [review]

**Review by morvencao — APPROVED** [review]

**Comment by zirain** [comment]
@Kampe can run `make gen` local to fix the CI, it will update `/sample/addons/grafana.yaml`

**Comment by ericvn** [comment]
@Kampe can run make gen local to fix the CI, it will update /sample/addons/grafana.yaml

**Comment by ericvn** [comment]
Likely need to rebase this PR. Trying to run `make gen` locally yields no changes.

**Pull request closed without merging**